packages/utils/src/version.ts

```diff
@@ -0,0 +1 @@
+export const SDK_VERSION = '8.4.0';
```

Comment:
this should be 8.5.0 already ;)

```suggestion
export const SDK_VERSION = '8.5.0';
```

Reply:
Also, can we actually make this the source of truth and have core re-export this? As core depends on utils already...?

Reply:
yeah that's a good point, will do!